src/pages/index.js

```diff
@@ -2,97 +2,224 @@ import "./index.css"; // add import of the main stylesheets file
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import {
+  avatarButton,
   validationSettings,
-  initialCards,
-  editProfileForm,
-  addCardForm,
-  cardListElement,
-  editProfileModal,
-  addCardModal,
-  previewImageModal,
-  editProfileButton,
+  profileNameInput,
+  profileDescriptionInput,
   addCardButton,
-  profileName,
-  profileDescription,
-  cardSelector,
+  editProfileButton,
+  selectors,
+  apiInfo,
 } from "../utils/constants.js";
 import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 
-// Initial start-up
+// Message to Tripleten reviewer: I'm so sorry for the terrible code.
+// I have really been struggling with this project and I dont know what I'm doing wrong
+// I keep getting an error in Popup.js, but I don't know whats wrong with it
+// Thank you for your patience
 
-const cards = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      cards.addItem(renderCard(cardData));
-    },
-  },
-  ".content__list"
-);
+// Initial start-up
+const { apiToken, apiUrl } = apiInfo;
 
-cards.renderItems();
+let userId = null;
+let section = null;
 
-// Validation
+const api = new Api({
+  baseUrl: apiUrl,
+  headers: {
+    authorization: apiToken,
+    "Content-Type": "application/json",
+  },
+});
 
-const editProfileFormValidator = new FormValidator(
-  validationSettings,
-  editProfileForm
+const userInfo = new UserInfo(
+  selectors.userNameSelector,
+  selectors.userDescriptionSelector,
+  selectors.userAvatarSelector
 );
-const addCardFormValidator = new FormValidator(validationSettings, addCardForm);
-editProfileFormValidator.enableValidation();
-addCardFormValidator.enableValidation();
-addCardFormValidator.resetButtonState();
-
-// Popup setup
 
-const userInfo = new UserInfo(profileName, profileDescription);
-
-const editProfileFormPopup = new PopupWithForm(
-  editProfileModal,
-  (inputValues) => {
-    userInfo.setUserInfo(inputValues.name, inputValues.description);
+const handleDeleteClick = (card) => {
+  confirmationModal.setSubmitAction(() => {
+    confirmationModal.setButtonText(true);
+    api
+      .deleteCard(card._cardId)
+      .then(() => {
+        card.deleteCard();
+        confirmationModal.close();
+      })
+      .catch((err) => {
+        console.error(err);
+      })
+      .finally(() => confirmationModal.setButtonText(false));
+  });
+  confirmationModal.open();
+};
+
+const handleLikeClick = (card) => {
+  if (card._isLiked) {
```

Comment:
Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.